ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__ppc64le-nightly.yaml

```diff
@@ -88,6 +88,21 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: ipi-powervs
+- as: powervs-ipi-e2e-test-qe-destructive
```

Comment:
Would you mind to use name "powervs-ipi-f7-destructive", so we have the same name style as the others ?